include/open62541/client.h

```diff
@@ -279,6 +285,14 @@ UA_Client_connectUsername(UA_Client *client, const char *endpointUrl,
     cc->userIdentityToken.content.decoded.data = identityToken;
     return UA_Client_connect(client, endpointUrl);
 }
+#ifdef UA_ENABLE_ENCRYPTION
+/* Connect to the server and create+activate a Session with the given X.509 certificate.
+ * This first set the X509IdentityToken in the client config and
+ * then calls the regular connect method. */
+UA_StatusCode UA_EXPORT
+UA_Client_connectCertificate(UA_Client *client, const char *endpointUrl,
```

Comment:
For logging in with username/pw we have two options:
1. Use UA_Client_connectUsername
2. Set up the userIdentityToken in the config and connect with UA_Client_connect

The same should possible for certificate authentication.
In the future we will have parsing of the config from some .ini file (JSON).

So I would prefer a public convenience function to set up the client config with cert and key.
And have UA_Client_connectCertificate first set up the config and then call into UA_Client_connect.